bigcache_test.go

```diff
@@ -1132,3 +1132,29 @@ func TestCache_RepeatedSetWithBiggerEntry(t *testing.T) {
 	}
 
 }
+
+// TestBigCache_allocateAdditionalMemoryLeadPanic
+// The new commit 16df11e change the encoding method,it can fix issue #300
+func TestBigCache_allocateAdditionalMemoryLeadPanic(t *testing.T){
+	t.Parallel()
+	cache, _ := NewBigCache(Config{
+		Shards:       1,
+		LifeWindow:   3 * time.Second,
+		MaxEntrySize: 52,
+	})
+	cache.Set("a",  blob(0xff, 235))
+	time.Sleep(2 * time.Second)
```

Comment:
We should not sleep in tests. Please use `mockedClock`

Reply:
I replaced time.Sleep with mockedClock.Please take a look